libs/platform/src/lib/components/link/link.component.html

```diff
@@ -0,0 +1,20 @@
+<a
+    #link
+    fd-link
+    [ngClass]="'fdp-link-truncate__txt'"
+    [attr.id]="id"
+    [attr.href]="href"
+    [attr.target]="target ? target : null"
+    [disabled]="disabled ? disabled : null"
+    [emphasized]="emphasized ? emphasized : null"
+    [inverted]="inverted ? inverted : null"
+    [title]="title ? title : null"
+    [attr.aria-label]="title ? title : null"
+    [attr.hreflang]="hreflang ? hreflang : null"
+    [attr.download]="download ? download : null"
+    [attr.media]="media ? media : null"
+    [attr.rel]="rel ? rel : null"
```

Comment:
I don't think using the ternary operator is the best approach here. You can set default values in the ts file and then have something like: 
`[disabled]="disabled"`

Reply:
simplified now.